src/Raven.Server/Web/System/TcpConnectionInfoHandler.cs

```diff
@@ -132,8 +132,19 @@ public static bool Authenticate(HttpContext httpContext, ServerStore serverStore
                         if (serverStore.Cluster.TryReadPullReplicationDefinition(database, remoteTask, context, out var pullReplication))
                         {
                             var cert = httpContext.Connection.ClientCertificate;
-                            if (pullReplication.CanAccess(cert?.Thumbprint))
-                                return true;
+                            if (pullReplication.Certificates != null && pullReplication.Certificates.Count > 0)
```

Comment:
Do we want that? Shouldn't we migrate data?

Reply:
That gets quite complex, unfortunately. Easier to just maintain compatability